hathor/metrics.py

```diff
@@ -254,8 +276,37 @@ def set_stratum_data(self) -> None:
         self.blocks_found = blocks_found
         self.estimated_hash_rate = estimated_hash_rate
 
+    def collect_peer_connection_metrics(self) -> None:
+        """Collect metrics for each connected peer.
+            The list is cleated every time to avoid memory leak.
```

Comment:
Typo: cleared*

Reply:
Fixed in https://github.com/HathorNetwork/hathor-core/pull/467/commits/fbc0453ccbcbed471d13cb3f66d7481a8666b8e7